<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acctOptsScreen</class>
 <widget class="QMainWindow" name="acctOptsScreen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="acctDtlsButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Acct. Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transferButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkDepositButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Check Deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateAcctButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Update Acct. Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="acctOptionsText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>-2</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Account Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="acctOptsMemberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Member:</string>
    </property>
   </widget>
   <widget class="QLabel" name="acctOptsMemName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>218</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Member Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAcctButton">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>3</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 0px solid white;
	color: &quot;red&quot;
}</string>
    </property>
    <property name="text">
     <string>Delete Acct</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
